src/themes/teams/components/Divider/dividerVariables.ts

```diff
@@ -0,0 +1,21 @@
+export interface IDividerVariables {
+  [key: string]: string | number
```

Comment:
Why pin an unknown key to a specific type? Mostly just curious.

Reply:
Just had to pick something and this seemed reasonable.  This is essentially a "first guess" at what acceptable style variable values would be.  Very much open to change and iteration.

We're targeting simple and flat variable objects.  I can say I don't think we should allow arrays and objects and such.  The theming system is pretty robust as it is, keeping variables simple will help make themes more usable.